web/src/js/components/Header/MainMenu.jsx

```diff
@@ -17,14 +17,18 @@ export default function MainMenu() {
     )
 }
 
+export function setIntercept(intercept) {
+    updateSettings({ intercept })
+}
+
 const InterceptInput = connect(
     state => ({
         value: state.settings.intercept || '',
         placeholder: 'Intercept',
         type: 'pause',
         color: 'hsl(208, 56%, 53%)'
     }),
-    { onChange: intercept => updateSettings({ intercept }) }
+    { onChange: setIntercept }
```

Comment:
Here we need to extract the anonymous function, otherwise this line could not be covered.